src/nl/hannahsten/texifyidea/project/TeXiFyProjectViewNodeDecorator.kt

```diff
@@ -27,9 +28,12 @@ class TeXiFyProjectViewNodeDecorator : ProjectViewNodeDecorator {
             }
         }
 
-        val icon = Magic.Icon.fileIcons[extension.toLowerCase()] ?: return
-
-        presentationData.setIcon(icon)
+        // Allow Material design plugins to take over the icons
+        // For file types registered in plugin.xml this happens automatically
+        if (PluginManager.getLoadedPlugins().none { it.name.contains("Material") }) {
```

Comment:
Why not filter by `"Icon"` instead of by `"Material"`? (Maybe `"Theme"` as well?) That would work for the Atom Material Icons plugin...

Reply:
But not for the `Material Theme UI` plugin?

Reply:
That's why I suggested to include `Theme` as well. Both make more sense than just `Material`.

Reply:
Ah, you mean an 'or'. Doesn't work: matches Windows 10 Light Theme plugin which is bundled.

Reply:
Yes. Oh that's shit.